content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -104,6 +104,10 @@ Right after `kubeadm init` there should not be any pods in these states.
   likely that the Pod Network solution that you installed is somehow broken. You
   might have to grant it more RBAC privileges or use a newer version. Please file
   an issue in the Pod Network providers' issue tracker and get the issue triaged there.
+- If you install old version of docker, lower than 1.12.1, please remove the 'MountFlags=slave' option
```

Comment:
`If you install old version of docker, lower than 1.12.1, please remove the 'MountFlags=slave' option`
-> 
`If you install a version of Docker older than 1.12.1, please remove the 'MountFlags=slave' option`

Reply:
This is a little weird to add instructions for 1.12.1 when it's not supported and you can't even obtain it on most distros anymore.

Reply:
Use backticks (not single quote) for code elements. Also, our style guide advises against using "please".

If you install a version of Docker older than 1.12.1, remove the `MountFlags=slave` option.